README.md

```diff
@@ -408,14 +408,7 @@ and CUDNN7 installations on your machine (for example, preinstalled on your
 cloud VM), you can run
 
 ```bash
-# install jaxlib
-PYTHON_VERSION=cp37  # alternatives: cp36, cp37, cp38
-CUDA_VERSION=cuda100  # alternatives: cuda100, cuda101, cuda102, cuda110
-PLATFORM=manylinux2010_x86_64  # alternatives: manylinux2010_x86_64
-BASE_URL='https://storage.googleapis.com/jax-releases'
-pip install --upgrade $BASE_URL/$CUDA_VERSION/jaxlib-0.1.52-$PYTHON_VERSION-none-$PLATFORM.whl
-
-pip install --upgrade jax  # install jax
+pip install --upgrade jax jaxlib==0.1.52+cuda110 -f https://storage.googleapis.com/jax-releases/jax_releases.html
```

Comment:
You should wait until we do a jaxlib release to make the README change.

Reply:
Why? I manually updated the already-released wheels so this command will still work. I think it's better to do this at all once if we can, so we don't forget to update the README later and break the install instructions.

Reply:
Ok, that's fine then. I didn't know you had rebuilt the existing release. (Although it still may avoid some versioning confusion if we just make a new release.)

Reply:
I didn't rebuild them, I only renamed them (and kept the existing ones as well), but I think that should be enough... it's possible the version reported by `jaxlib.__version__` will include the `+cudaXXX` once we rebuild, which it doesn't now, but I think that's fine. I can cut a new jaxlib release after this changes goes in to verify it all works and avoid any potential issues.